docs/system_configuration.md

```diff
@@ -21,64 +21,101 @@ order: 3
 {% raw %}
 
 ## Changes Since 0.5.0
-This configuration has been split into two different files. One file is for [definitions](/definitions) of various Bosun sections such as alerts, templates, notifications, etc. The definitions file can be edited and [saved via Bosun's Rule Editor UI](/usage#definition-rule-saving) when [the `EnableSave` setting](/system_configuration#enablesave) has been enabled. 
+Since 0.5.0, the config has been split into two different files.
 
-This was done because the definitions can now be reloaded without restarting the Bosun process. This also means that users can edit alerts directly in the UI.
+### System
+System config is documented on the rest of this page, and has been moved
+into `bosun.toml` unless overriden on the command line. Changes to these
+settings require a Bosun restart in order to take effect. The TOML
+format is documented [here](https://github.com/toml-lang/toml).
 
-System configuration has been moved into a new file. Settings in this file require that Bosun be restarted. The new file format is in [TOML](https://github.com/toml-lang/toml). This page documents this new system configuration. 
+The [example config
+file](https://github.com/bosun-monitor/bosun/blob/master/cmd/bosun/bosun.example.toml)
+is a good place to start when writing a configuration. NB The file does
```

Comment:
NB is a bit obscure. CSS has support for a warning css block which will draw more attention. For example:

```

<div class="admonition">
<p class="admonition-title">Tip</p>
<p>This file does not ...</p>
</div>
```